pulsemeeter/__main__.py

```diff
@@ -431,26 +443,32 @@ def main():
 
     #none: Start Server (if not already started) and open window 
     if len(sys.argv) == 1:
-        trayonly = False
+        if isserver: start_server(server)
```

Comment:
why start server here? what was the issue with the old version?

Reply:
I thought when you run pulsemeeter through dmenu or something like this and the server is not already started it should start it. Or what do you mean?

Reply:
the startapp function already started everything, you just moved it